requirements.txt

```diff
@@ -1,3 +1,4 @@
-git+https://github.com/bullish-exchange/python-signer
+eosio-signer@git+https://github.com/bullish-exchange/python-signer
 websocket-client==1.3.1
 requests==2.25.1
+python-dotenv
```

Comment:
python-dotenv for `load_dotenv()` function

Reply:
Can u add in the version?

Reply:
let me fix the version for all packages in requirements.txt.

It's a great idea. thanks